Sources/PointFree/Bootstrap.swift

```diff
@@ -20,6 +20,7 @@ private let stepDivider = print(message: "  -----------------------------")
 private let connectToPostgres =
   print(message: "  ⚠️ Connecting to PostgreSQL...")
     .flatMap { _ in AppEnvironment.current.database.migrate() }
+    .catch { print(message: "  ❌ Error! \($0)").flatMap(const(throwE($0))) }
```

Comment:
I was in a weird state locally. This is nice to have tho!

Reply:
Oh nice!